project/AppDependencies.scala

```diff
@@ -9,14 +9,17 @@ object AppDependencies {
 
   val compile = Seq(
     "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % "5.16.0",
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"         % "0.56.0",
+    "uk.gov.hmrc"             %% "simple-reactivemongo"       % "8.0.0-play-28",
```

Comment:
We're advised to use `hmrc-mongo` in preference to `simple-reactivemongo`
https://confluence.tools.tax.service.gov.uk/display/TEC/2021/03/04/HMRC+Mongo+is+now+available